sympy/solvers/tests/test_ode.py

```diff
@@ -1075,6 +1075,13 @@ def test_classify_sysode():
 
 def test_solve_ics():
     # Basic tests that things work from dsolve.
+    assert(dsolve(f(x).diff(x)*(f(x).diff(x, 2)-x), ics={f(0):0, f(x).diff(x).subs(x,0):0})) \
```

Comment:
The comment above doesn't match the tests you are adding. Can you move the new tests below and add a comment explaining that they are testing trickier cases.